app/views/users/show.html.erb

```diff
@@ -1 +1,23 @@
-<h1>Show individual user</h1>
\ No newline at end of file
+<section class = 'user-section'>
+  <div class = 'user-image'>
+         <img src= #{@user.photo}></img>
+  </div>
+  <div class = 'user-content'>
+    <div>
+      <h2><%= @user.name %></h2>
+    </div>
+
+    <div>
+      <p class = 'post-num'>Number of post: <%= @user.post_counter%></p>
+    </div>
+  </div>
+</section>
+
+<% if @user.bio.present? %>
+	<div class = 'user-bio'>
+    <h3>Bio</h3>
+    <p><%= @user.bio %></p>
+  </div>
+<% end %>
+<%= render partial: 'posts/post', collection: @post %> 
+<%= link_to "See all post", user_posts_path(@user.id), class: 'see-posts-button' %>
```

Comment:
- [ ] <b>REQUIRED:</b> The project requires to render the recent posts of the user here. Kindly do so to comply with the project requirements.

Reply:
@rukundoeric,

Thank you for the feedback.

This is exactly what I have done here. Please check the image below:

![image](https://user-images.githubusercontent.com/10332499/155314370-41fc030f-369e-4e84-af3d-5f45748861f8.png)

Reply:
@rukundoeric ,
Kindly reply to the message with the way forward. Thank you.